flash/core/data/process.py

```diff
@@ -569,6 +569,11 @@ class Deserializer(Properties):
     def deserialize(self, sample: Any) -> Any:  # TODO: Output must be a tensor???
         raise NotImplementedError
 
+    @property
+    @abstractmethod
+    def example_input(self) -> str:
+        raise NotImplementedError
```

Comment:
abstract wont raise anything it has to be implemented

Reply:
Fixed :smiley: